docs/source/03_tutorial/04_create_pipelines.md

```diff
@@ -795,7 +795,7 @@ def create_pipelines(**kwargs) -> Dict[str, Pipeline]:
 
 This decorator is commonly used and Kedro already includes it as a built-in decorator called `kedro.pipeline.decorators.log_time`.
 
-Another built-in decorator is `kedro.pipeline.decorators.mem_profile`, which will log the maximum memory usage of your node.
+Another extra decorator is `kedro.extras.decorators.mem_profile`, which will log the maximum memory usage of your node.
```

Comment:
I think this should actually be `kedro.extras.decorators.memory_profiler.mem_profile`.

Reply:
Thanks @mzjp2 corrected in new commit